Examples/MAX32520/SPI_MasterSlave/main.c

```diff
@@ -0,0 +1,170 @@
+/**
+ * @file    main.c
+ * @brief   SPI_MasterSlave Demo
+ * @details Shows Master loopback demo for QSPI0
+ *          Read the printf() for instructions
+ */
+
+/*******************************************************************************
+* Copyright (C) Maxim Integrated Products, Inc., All Rights Reserved.
+*
+* Permission is hereby granted, free of charge, to any person obtaining a
+* copy of this software and associated documentation files (the "Software"),
+* to deal in the Software without restriction, including without limitation
+* the rights to use, copy, modify, merge, publish, distribute, sublicense,
+* and/or sell copies of the Software, and to permit persons to whom the
+* Software is furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included
+* in all copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+* OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+* IN NO EVENT SHALL MAXIM INTEGRATED BE LIABLE FOR ANY CLAIM, DAMAGES
+* OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+* ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+* OTHER DEALINGS IN THE SOFTWARE.
+*
+* Except as contained in this notice, the name of Maxim Integrated
+* Products, Inc. shall not be used except as stated in the Maxim Integrated
+* Products, Inc. Branding Policy.
+*
+* The mere transfer of this software does not imply any licenses
+* of trade secrets, proprietary technology, copyrights, patents,
+* trademarks, maskwork rights, or any other form of intellectual
+* property whatsoever. Maxim Integrated Products, Inc. retains all
+* ownership rights.
+*
+******************************************************************************/
+
+/***** Includes *****/
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "gpio.h"
+#include "led.h"
+#include "mxc_device.h"
+#include "mxc_pins.h"
+#include "nvic_table.h"
+#include "pb.h"
+#include "spi.h"
+
+/***** Definitions *****/
+#define DATA_LEN 1024 // Words
+#define DATA_SIZE 8
+#define VALUE 0xFF
+#define SPI_SPEED 100000 // Bit Rate
```

Comment:
@Jacob-Scheiffler Have you tested this at higher speeds?  

On our other loopback examples there's a threshold where the example will fail, and we've thrown in a comment so customers are aware it's a limitation of the loopback setup and not the peripheral hardware.

I suspect there is a max `SPI_SPEED` for this example as well